pkg/runtime/k0s/common.go

```diff
@@ -15,8 +15,11 @@
 package k0s
 
 const (
-	GetK0sVersionCMD = "k0s version"
+	DefaultAdminConf = "/var/lib/k0s/pki/admin.conf"
 
-	SSHKeyGenCMD    = "ssh-keygen -f /root/.ssh/id_rsa -P \"\""
-	SSHCopyIDPrefix = "ssh-copy-id -f "
+	DefaultK0sConfigPath     = "/etc/k0s/k0s.yaml"
+	DefaultK0sWorkerJoin     = "/etc/k0s/worker"
```

Comment:
What does this const value mean? Could you add more comment for it? Actually I am confused that you add a `Join` in the const value name? And the detailed value, I think, is exactly a path, or config or something related. However, `Join` is verb, quite misleading. @starComingup

Reply:
This const would be k0s master/node join token path, but when I named it `DefaultK0sworkerJoinToken = "xxxx" `. The github action go security warned me don't use `Token` as a suffix or prefix, it would be checked by regex. So i am in a dilemma @allencloud